x-pack/solutions/security/plugins/security_solution/public/asset_inventory/hooks/use_asset_inventory_data_table/use_data_view.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { useQuery } from '@tanstack/react-query';
+import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+
+export const useDataView = (indexPattern: string) => {
```

Comment:
`x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/hooks/use_data_view.ts`

can this be used instead of creating a new hook?

Reply:
Not for 9.0 as `x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/hooks/use_data_view.ts` relies on having setup, that will create the Dataview, so this is a temporary hook, I will add a comment explaining that!